apps/studio/src/components/TabQueryEditor.vue

```diff
@@ -778,6 +779,30 @@ import { FavoriteQuery } from '@/common/appdb/models/favorite_query'
       },
       fakeRemoteChange() {
         this.query.text = "select * from foo"
+      },
+      getColumnsForAutocomplete() {
+        const cm = this.editor.getValue() 
+        if (cm.toLowerCase().search(/[from(\s)?|join(\s)?]/g) === -1) return
+        const triggerWords = ['join', 'from']
+        const allTables = this.hintOptions
+        const cmValue = cm.replace(/[;[\]'"]/g, '').replace(/\r?\n|\r/g, ' ').split(' ').filter(word => word !== '')
+        const tablesToFind = cmValue
+          .reduce((acc, word, index, arr) => {
+            if (index === arr.length) return acc
+
+            if (triggerWords.includes(word.toLowerCase())) {
+              // will need to clean up the word if it's wrapped up in stuff
+              if (allTables.tables[arr[index + 1]]?.length === 0){
+                acc.push(arr[index + 1])
+              }
+            }
+            return acc
+          }, [])
+          .forEach(async(table) => {
+            const tableToFind = this.tables.find(t => t.name === table)
```

Comment:
Need to filter by schema here too

Reply:
I don't believe schema is passed through sql hint at this time, so would need to add that. If you'd like me to for this release I certainly can.

Reply:
@wmontgomery Well you still need to filter to make sure the table is either in the default schema, or has no schema no?